test/extensions/filters/http/rbac/rbac_filter_test.cc

```diff
@@ -92,6 +109,9 @@ TEST_F(RoleBasedAccessControlFilterTest, Denied) {
   EXPECT_EQ(Http::FilterHeadersStatus::StopIteration, filter_.decodeHeaders(headers_, true));
   EXPECT_EQ(1U, config_->stats().denied_.value());
   EXPECT_EQ(1U, config_->stats().shadow_allowed_.value());
+
+  EXPECT_EQ("200", (*metrics_.mutable_fields())["shadow_response_code"].string_value());
```

Comment:
`metrics_.fields()["shadow_response_code"]`? ditto for below

Reply:
updated.